crates/next-core/js/src/ipc/index.ts

```diff
@@ -142,3 +142,39 @@ export const IPC = createIpc<unknown, unknown>(parseInt(PORT, 10));
 process.on("uncaughtException", (err) => {
   IPC.sendError(err);
 });
+
+const improveConsole = (name: string, stream: string, addStack: boolean) => {
+  // @ts-ignore
+  const original = console[name];
+  // @ts-ignore
+  const stdio = process[stream];
+  // @ts-ignore
+  console[name] = (...args: any[]) => {
+    stdio.write(`TURBOPACK_OUTPUT_B\n`);
+    original(...args);
+    if (addStack) {
+      const stack = new Error().stack?.replace(/^.+\n.+\n/, "") + "\n";
```

Comment:
Should we use a structured stack here?

Reply:
Yep, we should do that in the next iteration when you actually do something valueable with the stack trace.